spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'User Post Index Page', type: :feature do
+  fixtures :users, :posts
+
+  before do
+    @user = users(:john_doe)
+    @posts = @user.posts.page(1)
+    visit user_posts_path(@user)
+  end
+
+  it 'I can see the user\'s profile picture' do
+    expect(page).to have_selector("img[src='#{@user.photo}']")
+  end
+
+  it 'I can see the user\'s username' do
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'I can see the number of posts the user has written' do
+    expect(page).to have_content("Number of Posts: #{@user.posts.count}")
+  end
+
+  it 'I can see a post\'s title' do
+    post = @posts.first
+    visit user_post_path(@user, post)
+    expect(page.text.downcase).to include(post.title.downcase)
+  end
+
+  it 'I can see some of the post\'s body' do
+    post = @posts.first
+    visit user_post_path(@user, post)
+    expect(page).to have_selector('p', text: post.text)
+  end
+
+  it 'I can see the first comments on a post' do
+    post = @posts.first
+    visit user_post_path(@user, post)
+    post.recent_comments.each do |comment|
+      expect(page).to have_content(comment.text)
+    end
+  end
+
+  it 'I can see how many comments a post has' do
+    post = @posts.first
+    visit user_post_path(@user, post)
+    expect(page).to have_content("Comments: #{post.comments.count}")
+  end
+
+  it 'I can see how many likes a post has' do
+    post = @posts.first
+    visit user_post_path(@user, post)
+    expect(page).to have_content(post.likes.count.to_s)
+  end
```

Comment:
- [ ] In these test blocks you are visiting the post show page, not the required post index page for this test file. Kindly visit the correct routes and ensure no test specification is missing.